lib/datasource/rubygems/get.ts

```diff
@@ -29,12 +32,95 @@ export async function fetch(
   return response.body;
 }
 
+export async function fetchDependencies(
+  dependencies: string[],
+  registry: string
+): Promise<any> {
```

Comment:
Do not use `any` create interfaces and add to  `./types.ts`

Reply:
I used `any` here because the existing function (the `fetch` above) uses it. Do you mean I should avoid `any` in new codes? I'll try to then.

Reply:
92a75a9